### PR TITLE
[Logs] Edits to HTTP requests, FW events fields.

### PR DESCRIPTION
**Pull request by angelampcosta**

### CODE REVIEW

File: content/logs/reference/log-fields/zone/firewall_events.md [hunk 1]
```diff
@@ -41,6 +41,6 @@ The descriptions below detail the fields available for `firewall_events`.
...
 | RuleID | The Cloudflare security product-specific RuleID triggered by this request | string |
-| Source | The Cloudflare security product triggered by this request. <br />Possible sources are <em>unknown</em> \| <em>asn</em> \| <em>country</em> \| <em>ip</em> \| <em>iprange</em> \| <em>securitylevel</em> \| <em>zonelockdown</em> \| <em>waf</em> \| <em>firewallrules</em> \| <em>uablock</em> \| <em>ratelimit</em> \| <em>bic</em> \| <em>hot</em> \| <em>l7ddos</em> \| <em>validation</em> \| <em>botfight</em> \| <em>apishield</em> \| <em>botmanagement</em> \| <em>dlp</em> \| <em>firewallmanaged</em> \| <em>firewallcustom</em> | string |
+| Source | The Cloudflare security product triggered by this request. <br />Possible sources are <em>unknown</em> \| <em>asn</em> \| <em>country</em> \| <em>ip</em> \| <em>iprange</em> \| <em>securitylevel</em> \| <em>zonelockdown</em> \| <em>waf</em> \| <em>firewallrules</em> \| <em>uablock</em> \| <em>ratelimit</em> \| <em>bic</em> \| <em>hot</em> \| <em>l7ddos</em> \| <em>botfight</em> \| <em>apishield</em> \| <em>botmanagement</em> \| <em>dlp</em> \| <em>firewallmanaged</em> \| <em>firewallcustom</em> | string |
```
Comment (marciocloudflare) [inline content/logs/reference/log-fields/zone/firewall_events.md:44]: Any reason why these values are italic? Normally we use italics for drop-down menu options. Should these be monospaced instead?

File: content/logs/reference/log-fields/zone/http_requests.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ The descriptions below detail the fields available for `http_requests`.
...
+| FirewallMatchesSources | The firewall products that matched the request. The same product can appear multiple times, which indicates different rules or actions that were activated. The RuleIDs can be found in FirewallMatchesRuleIDs, the actions can be found in FirewallMatchesActions. The length of the array is the same as FirewallMatchesRuleIDs and FirewallMatchesActions. <br />Possible sources are <em>unknown</em> \| <em>asn</em> \| <em>country</em> \| <em>ip</em> \| <em>ipRange</em> \| <em>securityLevel</em> \| <em>zoneLockdown</em> \| <em>waf</em> \| <em>firewallRules</em> \| <em>uaBlock</em> \| <em>rateLimit</em> \| <em>bic</em> \| <em>hot</em> \| <em>l7ddos</em> \| <em>botFight</em> \| <em>apiShield</em> \| <em>botManagement</em> \| <em>dlp</em> \| <em>firewallManaged</em> \| <em>firewallCustom</em> | array[string] |
```
Comment (marciocloudflare) [inline content/logs/reference/log-fields/zone/http_requests.md:67]: Same question here regarding italics

**Review by marciocloudflare — APPROVED** [review]

File: content/logs/reference/log-fields/zone/firewall_events.md (hunk 1 above)
Comment (angelampcosta) [reply content/logs/reference/log-fields/zone/firewall_events.md:44]: I don't think so. Eventually we could change all of them.

Comment (pedrosousa) [reply content/logs/reference/log-fields/zone/firewall_events.md:44]: This would have to be changed in the team's field list generation script.

**Pull request merged**